USAGE_desktop.md

```diff
@@ -646,3 +647,15 @@ gfxrecon.py capture -o vkcube.gfxr vkcube
 
 On Windows, after installing Python3, be sure to associate the `.py` file extension with
 the Python3 interpreter before you run the script.
+
+### Options Common To all Tools
+
+If the environment variable `GFXRECON_NO_DEBUG_POPUP` has any non-empty
+value (e.g. "1" or "true" or "foo") when running any of of the file processing
```

Comment:
Excuse me.  A silly question. Why is `foo`? What is this?

Reply:
Not silly at all.  It's an awkward option.  The other options are compared against "1" or "true" or "foo".  In order to reduce the number of operations and thus the chance of failure or exception I just turn on the option if `GFXRECON_NO_DEBUG_POPUP` exists at all.  Should I just drop the statement in parentheses?

Reply:
On Windows, there's no way to unset an environment variable, only a way to set it to empty, right?  So we definitely need some check on the value of the environment variable.

I've also seen situations where someone accidentally (on Windows) set an environment variable to nothing but blank spaces, which was treated as `true` because it was not empty...   And I've seen confusion when someone `set GFXRECON_NO_DEBUG_POPUP=0` and expected it to behave as unset...

For simple comparisons, I prefer that the value is `false` if:
- the environment variable doesn't exist
- the environment variable is empty
- the environment variable is just spaces
- the environment variable contains a number that evaluates to 0
and is `true` in all other cases.

If we want to get fancy, I'd add to the `false` evaluation:
- case-insensitive `no`, `off`, and `false`

Reply:
As I understand it, an environment variable can be unset on Windows (`ERROR_ENVVAR_NOT_FOUND`) and that's different from "" (succeeded but length 0).

I probably could extend this in a simple way so that the variable is only true for "true", "True", "TRUE", or a number not 0.

Reply:
I got it. `foo` means anything else.

Reply:
We might need to add `number not 0`.

Reply:
I think this does all the above.  Tested it in December.